G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelExport/ShouldExportFile_WhenExcelExportCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,7 @@
+﻿excel.open
+
+excel.setvalue value 1 row 1 colindex 1
+♥path = ‴♥environment⟦TEMP⟧\testexcelsheet.pdf‴
+excel.export path ♥path
+file.delete ♥path timeout 2
+excel.close
```

Comment:
where is the verification that the file was exported correctly?

Reply:
I don't know if y'all are aware that the file.delete command never throws any exceptions ;) If the file exists, it deletes it, if it doesn't exist, simply nothing happens.